torch/csrc/autograd/record_function.h

```diff
@@ -83,6 +85,8 @@ struct TORCH_API StringView {
 // Soft limit on the number of callbacks to use;
 constexpr std::size_t kSoftLimitCallbacks = 32;
 
+typedef c10::SmallVector<uint64_t, kSoftLimitCallbacks> handles_vector;
```

Comment:
this is actually 256 bytes now - make the soft limit smaller? (usually we'd likely have <5 callbacks, so maybe 8?)